client/src/components/Reviews.jsx

```diff
@@ -1,5 +1,82 @@
 import React from 'react';
 import StarRatingComponent from 'react-star-rating-component';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  font-size: 14px;
+  font-family: "Lato", "Helvetica", "sans-serif";
+`;
+
+const ReviewWrapper = styled.div`
+  margin-bottom: 1.5em;
+
+  :after {
+    content: "";
+    display: block;
+    clear: both;
+  }
+`;
+
+const Img = styled.img`
+  float: left;
```

Comment:
I don't think this will interfere with this project (only do this if you have time), but worth noting that a standard solve in this scenario would be using CSS flexbox patterns with media queries. Lets you have the page adapt to multiple screens more easily. Floats can end up doing nasty things in the case of adding more elements to the page: https://css-tricks.com/dont-overthink-flexbox-grids/